substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/thread/Target_java_lang_Thread.java

```diff
@@ -182,10 +186,14 @@ public final class Target_java_lang_Thread {
     @RecomputeFieldValue(kind = RecomputeFieldValue.Kind.Reset) //
     Object lockHelper;
 
-    @Inject @TargetElement(onlyWith = LoomJDK.class) //
+    @Inject @TargetElement(onlyWith = {HasSetExtentLocalCache.class, HasExtentLocalCache.class, LoomJDK.class}) //
     @RecomputeFieldValue(kind = RecomputeFieldValue.Kind.Reset) //
     Object[] extentLocalCache;
```

Comment:
There is no need to _inject_ differently-named fields based on the name in the JDK that we're building on. You can rename the field to `scopedValueCache` in either case and use it from `extentLocalCache` and `setExtentLocalCache` too.

Reply:
Done